app/views/projects/samples/transfers/create.turbo_stream.erb

```diff
@@ -0,0 +1,31 @@
+<% if message %>
+  <%= turbo_stream.update "transfer_alert" do %>
+    <%= viral_alert(type:, message:, classes: "mb-4 h-30 overflow-y-auto") do %>
+      <ul class="mt-1.5 ml-4 list-disc list-inside">
```

Comment:
**nitpick:** The `ul` is showing as white space if no errors exist.  Can you wrap the whole list in an if statement?
![image](https://github.com/phac-nml/irida-next/assets/11295750/eaab3ca3-98d3-4e35-a757-81b9c1da707b)

Reply:
Please see d7859500ab11fcce1d1b1262cc43cf0f07f9e25c.

Reply:
Left a small comment on the commit.

Reply:
Please see d908a6557eb5cbcbafedabd8d2c288f8b858f966.